src/core/main.js

```diff
@@ -462,9 +462,10 @@ for (const k in constants) {
  * ```
  *
  * Code placed in `setup()` will run once before code placed in
- * <a href="#/p5/draw">draw()</a> begins looping. If the
- * <a href="#/p5/preload">preload()</a> is declared, then `setup()` will
- * run immediately after <a href="#/p5/preload">preload()</a> finishes
+ * <a href="#/p5/draw">draw()</a> begins looping. When `setup()` is declared async, 
```

Comment:
How about a slightly more precise description:

"When `setup()` uses the `async` keyword (like this: `async function setup()` instead of `function setup()`)...."